src/forms/checkbox.test.jsx

```diff
@@ -61,14 +73,15 @@ describe('Checkbox', function() {
 		});
 
 		it('calls onChange and sets state when clicked', function() {
-			const component = mount(JSXcheckboxUnchecked);
+			fakeOnChange.mockClear();
+			const component = mount(JSXcheckboxUncheckedWithOnChange);
 			const checkbox = component.find('input');
 
 			expect(checkbox.props().checked).toBe(false);
 			checkbox.simulate('change', { target: { checked: true } });
 			expect(checkbox.props().checked).toBe(true);
+			expect(fakeOnChange).toHaveBeenCalledWith(true);
```

Comment:
I think we have been putting an `expect` before the call to `simulate`  to make sure the mock has not been called